jp2 driver missing?

Hi,
I get this error message when I try to open a jp2 file:
```
ERROR: GDALError (CE_Failure, code 4):
	`L2A_T29UMV_20160926T115352_B02_10m.jp2' not recognized as a supported file format.
```

How can I fix this? I use Linux Ubuntu 16.04. 
Thank you
Bjoern

I think it'd be good to support it in the BinaryBuilder.

Right now I'm working on getting a working GDAL 3.0 build here: https://github.com/JuliaGeo/GDALBuilder/tree/gdal3

Most platforms seem to be working, but on FreeBSD I get an error `configure: error: long long not found`, and on Windows it for some reason does not find the PROJ 6.1 installation from BinaryBuilder. (https://travis-ci.org/JuliaGeo/GDALBuilder)

Since adding more drivers will only reduce the chance of a successful build, I prefer to first get a working GDAL 3.0 build out before adding more drivers.

--- a/Project.toml
+++ b/Project.toml
@@ -13,7 +13,7 @@ PROJ_jll = "58948b4f-47e0-5654-a9ad-f609743f8632"
 [compat]
 CEnum = "0.2, 0.3, 0.4"
 GDAL_jll = "3.0.4"
-PROJ_jll = "6.3.0"
+PROJ_jll = "7.0.1"
 julia = "1.3"
 
 [extras]

--- a/test/drivers.jl
+++ b/test/drivers.jl
@@ -7,6 +7,7 @@ available_drivers = [
     "AAIGrid",
     "GPKG",
     "GTiff",
+    "JP2OpenJPEG",
     "MEM",
     "PCRaster",
     # libcurl raster drivers
@@ -44,12 +45,4 @@ available_drivers = [
 
 for drivername in available_drivers
     @test GDAL.gdalgetdriverbyname(drivername) != C_NULL
-end
-
-not_available_drivers = [
-    "JP2OpenJPEG",  # up next, see #64
-]
-
-for drivername in not_available_drivers
-    @test_broken GDAL.gdalgetdriverbyname(drivername) != C_NULL
 end

